frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -84,6 +86,55 @@ export default class Metadata extends React.Component<Props> {
         </Localized>;
     }
 
+    renderGroupComment(entity: Entity): React.Node {
+        if (!entity.group_comment) {
+            return null;
+        }
+
+        let comment = entity.group_comment;
```

Comment:
Since this is only used once, you could just use `entity.group_comment` directly. Also below for the Resource comment.